apps/backend/src/flat-files/tdi17/TDI17Details.ts

```diff
@@ -17,10 +17,10 @@ export interface ITDI17Details extends IFixedWidthRecord<ITDI17Details> {
   deposit_time: string;
   seq_no: string;
   location_desc: string;
-  deposit_amt_curr: number;
+  deposit_amt_curr: string;
```

Comment:
why is this changing to string?

Reply:
For ease of testing. This way we can ensure the correct number of characters in the string. I had planned to parse it while working with the data later, and to just keep this value exactly as is until we are ready to work with the data.